Apps/Common/src/Validations/VaccineStatusQueryValidator.cs

```diff
@@ -17,7 +17,7 @@
 namespace HealthGateway.Common.Validations
 {
     using FluentValidation;
-    using HealthGateway.Common.Data.Validation;
```

Comment:
I think this class was introduced in a previous PR but can this method be unit tested directly or is it being tested indirectly through a service?

Reply:
Added unit  tests